cryptodredge/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM nvidia/cuda:9.1-base
+
+RUN apt-get update > /dev/null \
+    && apt-get install wget \
```

It will prevent apt from asking for confirmation,
and then also remove apt's cache to decrease image size.

```suggestion
    && apt-get install wget -y \
    && rm -rf /var/lib/apt/lists/* && \
```